dao/src/main/java/com/redhat/ipaas/dao/manager/DataManager.java

```diff
@@ -140,13 +142,14 @@ public String generatePK(Cache<String, WithId> entityMap) {
 
     @SuppressWarnings("unchecked")
     public <T extends WithId> ListResult<T> fetchAll(Class<T> model, Function<ListResult<T>, ListResult<T>>... operators) {
-        Kind kind = Kind.from(model);
```

Comment:
Having a PR that includes unrelated changes to the pr title, albeit with details in commit messages, doesn't ​make reviews easy... 

Once we get post demo we're going to work out better ways to do this.